src/components/Kv/KvFlag2.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+	<div :class="`kv-flag kv-flag--${aspectRatio}`">
+		<div class="kv-flag__wrapper">
+			<template v-if="inlineSvg">
+				<component
+					:is="flagSVG"
+				/>
+			</template>
+			<template v-else>
+				<div
+					class="kv-flag__img-sprite"
+					:style="{ backgroundPositionY: spriteYPosition + '%' }"
+				></div>
+			</template>
+			<span class="show-for-sr">{{ countryName }}</span>
+		</div>
+	</div>
+</template>
+
+<script>
+import { getCodes, getNameByCode } from 'flag-icon-css';
+
+const COUNTRY_LIST = getCodes();
+const SPRITE_FLAG_WIDTH = 32; // Number of px wide that the sprite PNG is.
+
+export default {
+	props: {
+		/**
+		 * 2 letter ISO country code of the flag to show
+		* */
+		country: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * Aspect Ratio of the flag image
+		 * `4x3, 1x1`
+		* */
+		aspectRatio: {
+			type: String,
+			default: '4x3',
+		},
+		/**
+		 * Inline the flag SVG into the markup instead of using a background image sprite
+		 * Use when showing a small amount of flags per page or if you need a larger flag than 32px wide.
+		* */
+		inlineSvg: {
+			type: Boolean,
+			default: false
+		}
+	},
```

Comment:
I think it would be good to add a cache key to this component to help this render really fast on the server. Something like:
```
import getCacheKey from '@/util/getCacheKey';
...
serverCacheKey: props => getCacheKey(`KvFlag-${props.country}-${props.aspectRatio}-${props.inlineSvg}`),
```

Reply:
Ooh, great idea! 

> I think it would be good to add a cache key to this component to help this render really fast on the server. Something like:
> 
> ```
> import getCacheKey from '@/util/getCacheKey';
> ...
> serverCacheKey: props => getCacheKey(`KvFlag-${props.country}-${props.aspectRatio}-${props.inlineSvg}`),
> ```